atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -345,7 +345,7 @@ def get_logs(self):
             logfile = NamedTemporaryFile(prefix=self.build_id,
                                          suffix=".log",
                                          mode='wb')
-            logfile.write(logs.encode('UTF-8'))
+            logfile.write(logs)
```

Comment:
I believe this should be compatible with old and new versions of osbs-client.

It's just that we need this new version of atomic-reactor when using the new osbs-client in the builder image they are both in.